### PR TITLE
FT: Add signup and signin

### PR DESCRIPTION
**Pull request by thoth-ky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "cors": "^2.8.4",
+    "express": "^4.16.3",
```
Comment (clemwek) [inline package.json]: why do you need express?

**Review by clemwek — COMMENTED** [review]
Did you decide to work with vanilla react?

File: package.json (hunk 1 above)
Comment (thoth-ky) [reply package.json]: Don't currently need it. Will remove it

**Comment by thoth-ky** [comment]
Yeah. I've decided to use vanilla React

**Pull request merged**